lib/utils/cli/cli.go

```diff
@@ -93,10 +100,24 @@ func (r *CommandArgs) addRemoveFlags(ctx *kingpin.ParseContext) (flags []Flag) {
 			flags = append(flags, flag)
 		}
 	}
+	for _, flag := range r.FlagsToReplace {
```

Comment:
Maybe also add a test for the new attribute?

Reply:
Added a test case for flag replacement. I noticed that the replacement could alter the order of the flags provided. I've updated the implementation a bit in 4320279.